test/extended/cli/observe.go

```diff
@@ -68,11 +68,11 @@ var _ = g.Describe("[sig-cli] oc observe", func() {
 		g.By("argument templates")
 		out, err = oc.Run("observe").Args("services", "--once", "--all-namespaces", "--template='{ .spec.clusterIP }'").Output()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(out).To(o.ContainSubstring("172.30.0.1"))
+		o.Expect(out).To(o.Or(o.ContainSubstring("172.30.0.1"), o.ContainSubstring(":")))
```

Comment:
Could we include the full IPv6 address like they do for IPv4? I would guess it's predictable for IPv6 if it's predictable for IPv4

Reply:
Good point. Let me check if I can extract the IPv6.

Reply:
Unfortunately, there is no statically defined IPv6. It seems that cluster is configured for IPv4;

`apiVersion: v1
kind: Service
metadata:
  labels:
    external: service
  name: external
  resourceVersion: "1"
  uid: 19cff995-5546-11e5-9f57-080027c5bfa9
spec:
  clusterIP: 172.30.0.100
  ports:
  - nodePort: 0
    port: 443
    protocol: TCP
    targetPort: 443
  sessionAffinity: None
  type: ClusterIP
status:
  loadBalancer: {}`